display_list/skia/dl_sk_dispatcher.h

```diff
@@ -14,11 +14,20 @@
 namespace flutter {
 
 //------------------------------------------------------------------------------
-/// Can be fed to the dispatch() method of a DisplayList to feed the resulting
-/// rendering operations to an SkCanvas instance.
+/// @brief      A Dispatcher that feeds rendering operations to an SkCanvas.
 ///
-/// Receives all methods on Dispatcher and sends them to an SkCanvas
+/// This class is intended to be used as the developer-facing interface to Skia
```

Comment:
Similarly, this is *NOT* the developer-facing API.

DlOpReceiver is the internal API that only backends implement, and this is Skia's implementation of it.

Reply:
Simplified to point at the interface.